src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1179,6 +1220,11 @@ FixedwingAttitudeControl::task_main()
 				_actuators.control[actuator_controls_s::INDEX_THROTTLE] = _manual.z;
 			}
 
+			// Add feed-forward from roll control output to yaw control output
+			// This can be used to counteract the adverse yaw effect when rolling the plane
+			_actuators.control[actuator_controls_s::INDEX_YAW] += _parameters.roll_to_yaw_ff * math::constrain(
```

Comment:
It's not the same. The coordinated turn tries to balance the acceleration on a general level. This on the other hand is a very direct compensation of some aircraft physical properties comparable with control surface trims or angling the motor mount to compensate for side thrust.